library/extractor/src/main/java/com/google/android/exoplayer2/metadata/vorbis/PictureFrame.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.exoplayer2.metadata.vorbis;
+
+import static com.google.android.exoplayer2.util.Util.castNonNull;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import androidx.annotation.Nullable;
+import com.google.android.exoplayer2.MediaMetadata;
+import com.google.android.exoplayer2.metadata.Metadata;
+import java.util.Arrays;
+
+/** A picture parsed from a FLAC file. */
```

Comment:
I don't think this is correct any more?

It seems the [VorbisComment spec refers to a 'FLAC picture block'](https://wiki.xiph.org/VorbisComment#General_encoding.2Fdecoding), so maybe this javadoc should be updated to refer to that wording (rather than concretely stating it was parsed from  a FLAC file).

Reply:
Resolved in fc08f86.